### PR TITLE
Task 7

### PR DESCRIPTION
**Pull request by Damz1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -186,8 +186,56 @@ describe("/api/reviews/:review_id/comments", () => {
...
+      .send({
+        username: "David",
+        body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s w.",
+      })
+      .expect(406)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("not acceptable");
+      });
+  });
```
Comment (robert112233) [inline __tests__/app.test.js:279]: These are interesting tests that I haven't considered. However the tests for the core functionality are missing. They are listed below: 

Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist in the DB

**Review by robert112233 — CHANGES_REQUESTED** [review]
Good work, some missing tests in a comment below 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -186,8 +186,95 @@ describe("/api/reviews/:review_id/comments", () => {
...
+          comment_id: expect.any(Number),
+          body: "my posted comment",
+          review_id: expect.any(Number),
+          author: "mallionaire",
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("POST 406: should not post if missing comment", () => {
```
Comment (RichardNeat) [inline __tests__/app.test.js]: Just a 400 would be fine here, a 406 is a little more complicated and not quite right.

**Review by RichardNeat — APPROVED** [review]
Nice one David - just one minor comment in your test suite but no need to resubmit :)

**Pull request merged**